src/styles/dark-colors.css

```diff
@@ -1,47 +1,47 @@
-    --dark-border--COLOR-SUCCESS-DARK: #329e59;
-    --dark-bg--COLOR-INFO: #0245b2;
-    --dark-bg--COLOR-WARNING: #b9520a;
-    --dark-text--COLOR-WARNING: #f4802d;
-    --dark-border--COLOR-WARNING: #ad4d09;
-    --dark-bg--COLOR-DANGER: #a52525;
-    --dark-text--COLOR-DANGER: #d54545;
-    --dark-border--COLOR-DANGER: #932121;
-    --dark-bg--COLOR-SUCCESS: #0c8036;
-    --dark-text--COLOR-SUCCESS: #60f095;
-    --dark-border--COLOR-SUCCESS: #11b94f;
-    --COLOR-ONLINE: #0cdd59;
-    --dark-text--COLOR-BUSY: #ff1a1a;
-    --dark-bg--COLOR-OFFLINE: #60686c;
-    --dark-text--COLOR-SNIPPET-COMMENTS: #ffce6e;
-    --dark-text--COLOR-STRINGS: #a9f0ff;
-    --dark-text--COLOR-IMPORT: #ff941a;
-    --dark-bg--COLOR-BOX-SHADOW1: rgba(0, 0, 0, 0.2);
-    --dark-bg--COLOR-BOX-SHADOW2: rgba(0, 0, 0, 0.14);
-    --dark-bg--COLOR-BOX-SHADOW3: rgba(0, 0, 0, 0.12);
-    --dark-bg--COLOR-BOX-SHADOW4: rgba(40, 40, 74, 0.1);
-    --dark-bg--COLOR-BOX-SHADOW5: rgba(0, 0, 0, 0.08);
-    --dark-text--COLOR-ACTIVE: #1affff;
-    --dark-bg--COLOR-DISCOUNT-BG: #513100;
-    --dark-bg--COLOR-BOX-SHADOW-CARD: rgba(0, 0, 0, 0.56);
-}
\ No newline at end of file
+	--dark-bg--COLOR-WHITE: #181a1b;
+	--dark-text--COLOR-WHITE: #e8e6e3;
+	--dark-border--COLOR-WHITE: #303436;
+	--dark-text--COLOR-BLACK: #e8e6e3;
+	--dark-border--COLOR-BLACK: #8c8273;
+	--dark-bg--COLOR-BG: #212325;
+	--dark-bg--COLOR-OFFWHITE: #2a2d2f;
+	--dark-border--COLOR-OFFWHITE: #233656;
+	--COLOR-OFFBLACK: #272b2c;
+	--dark-bg--COLOR-PRIMARY: #1b4483;
+	--dark-text--COLOR-PRIMARY: #72abe2;
+	--dark-border--COLOR-PRIMARY: #2254a2;
+	--dark-bg--COLOR-PRIMARY-DARK: #12315f;
+	--dark-bg--COLOR-SECONDARY: #836423;
+	--dark-border--COLOR-SECONDARY: #7e6022;
+	--dark-bg--COLOR-SECONDARY-DARK: #56472a;
+	--dark-text--COLOR-SECONDARY-DARK: #cab892;
+	--dark-border--COLOR-SECONDARY-DARK: #8c7444;
+	--dark-bg--COLOR-TERTIARY: #144b57;
+	--dark-text--COLOR-TERTIARY: #a0dcea;
+	--dark-border--COLOR-SUCCESS-DARK: #329e59;
+	--dark-bg--COLOR-INFO: #0245b2;
+	--dark-bg--COLOR-WARNING: #b9520a;
+	--dark-text--COLOR-WARNING: #f4802d;
+	--dark-border--COLOR-WARNING: #ad4d09;
+	--dark-bg--COLOR-DANGER: #a52525;
+	--dark-text--COLOR-DANGER: #d54545;
+	--dark-border--COLOR-DANGER: #932121;
+	--dark-bg--COLOR-SUCCESS: #0c8036;
+	--dark-text--COLOR-SUCCESS: #60f095;
+	--dark-border--COLOR-SUCCESS: #11b94f;
+	--COLOR-ONLINE: #0cdd59;
+	--dark-text--COLOR-BUSY: #ff1a1a;
+	--dark-bg--COLOR-OFFLINE: #60686c;
+	--dark-text--COLOR-SNIPPET-COMMENTS: #ffce6e;
+	--dark-text--COLOR-STRINGS: #a9f0ff;
+	--dark-text--COLOR-IMPORT: #ff941a;
+	--dark-bg--COLOR-BOX-SHADOW1: rgba(0, 0, 0, 0.2);
+	--dark-bg--COLOR-BOX-SHADOW2: rgba(0, 0, 0, 0.14);
+	--dark-bg--COLOR-BOX-SHADOW3: rgba(0, 0, 0, 0.12);
+	--dark-bg--COLOR-BOX-SHADOW4: rgba(40, 40, 74, 0.1);
+	--dark-bg--COLOR-BOX-SHADOW5: rgba(0, 0, 0, 0.08);
+	--dark-text--COLOR-ACTIVE: #1affff;
+	--dark-bg--COLOR-DISCOUNT-BG: #513100;
+	--dark-bg--COLOR-CARD-BOX-SHADOW: rgba(0, 0, 0, 0.56);
```

Comment:
This is call good code structure using variable very well.

Reply:
Thanks, but will apply dark theme in next drafts.